acceptance-tests/pom.xml

```diff
@@ -0,0 +1,142 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.jenkins.plugins</groupId>
+    <artifactId>acceptance-tests</artifactId>
+    <name>Bitbucket Server Integration Plugin Acceptance Tests</name>
+    <description>Includes the integration/acceptance tests for the Bitbucket Server Jenkins integration plugin</description>
+    <packaging>jar</packaging>
+    <version>1.1.1-SNAPSHOT</version>
+
+    <properties>
+        <jenkins.version>2.176.1</jenkins.version>
+        <scribejava.version>6.8.1</scribejava.version>
+        <jackson.version>2.10.3</jackson.version>
+        <groovy.version>2.4.12</groovy.version>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.github.scribejava</groupId>
+            <artifactId>scribejava-apis</artifactId>
+            <version>${scribejava.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.scribejava</groupId>
+            <artifactId>scribejava-core</artifactId>
+            <version>${scribejava.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci</groupId>
+            <artifactId>acceptance-test-harness</artifactId>
+            <version>1.70</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.10</version>
```

Comment:
We should either move all versions as properties or define it inside dependency. I prefer defining them as property. With the module approach, you don't have to specify versions.

Reply:
> move all versions as properties

Is this what we do for _all_ dependencies elsewhere? I can see that for repeated versions (i.e. multiple dependencies of the same "family", e.g. all the Spring stuff) we use properties, but what's the benefit of using properties for every single dependency?

Reply:
Again, I think we should either change this everywhere (i.e. in the plugin pom), or leave as is, and I feel like it's outside the scope of this PR to change it everywhere.

Reply:
That makes sense. I'll change them here and will raise a separate PR for the ones in the plugin pom :+1: